modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2932,6 +2932,8 @@ curl --fail -u <my_username>:<my_password> -X POST "http://localhost:14240/gsqls
 
 This endpoint reports statistics of running queries of a graph: the query's request ID, start time, expiration time, and the REST endpoint's URL.
 
+This endpoint only shows queries sent to the requested node, not all queries in the graph.
+
```

Comment:
"Node" should be in contrast to "cluster", not "graph". 

"If you are running a TigerGraph cluster, this endpoint only shows queries running on the node to which the request is sent, not all running queries on the cluster."